kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -423,41 +422,29 @@ public void deactivateSubscription(Subscription subscription, DeactivationReason
     }
 
     @Override
+    @Cacheable("pack-type")
```

Comment:
As of now, I'm only caching domain entities. I imagine we'd want to address that if we were trying to cache non domain entities, right?